<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RulePopup</class>
 <widget class="QDialog" name="RulePopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RulePopup</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,4,1">
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font: 700 13pt &quot;Tahoma&quot;;</string>
     </property>
     <property name="text">
      <string>Luật nhân</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: 700 11pt &quot;Segoe UI&quot;;
color: lightgrey;</string>
         </property>
         <property name="text">
          <string>Original:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1,3">
         <item>
          <widget class="QLineEdit" name="original_start">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="focusPolicy">
            <enum>Qt::FocusPolicy::ClickFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 15pt &quot;Noto Sans SC&quot;;
background-color: transparent;
border-width: 0;
outline: none;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="original_placeholder">
           <property name="styleSheet">
            <string notr="true">font: 700 16pt &quot;Segoe UI&quot;;
border: 1px solid #0078d7;
border-radius: 15px;
padding: 0 2px;</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="original_end">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="focusPolicy">
            <enum>Qt::FocusPolicy::ClickFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 15pt &quot;Noto Sans SC&quot;;
background-color: transparent;
border-width: 0;
outline: none;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">font: 700 11pt &quot;Segoe UI&quot;;
color: lightgrey;</string>
         </property>
         <property name="text">
          <string>Translation:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,1,3">
         <item>
          <widget class="QLineEdit" name="translation_start">
           <property name="focusPolicy">
            <enum>Qt::FocusPolicy::ClickFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: transparent;
border-width: 0;
font: 12pt &quot;Tahoma&quot;;
padding: 6px 0;
outline: none;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="translated_placeholder">
           <property name="styleSheet">
            <string notr="true">font: 700 16pt &quot;Segoe UI&quot;;
border: 1px solid #0078d7;
border-radius: 15px;
padding: 0 2px;</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="translation_end">
           <property name="focusPolicy">
            <enum>Qt::FocusPolicy::ClickFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: transparent;
border-width: 0;
font: 12pt &quot;Tahoma&quot;;
padding: 6px 0;
outline: none;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="close_popup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 13pt;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
